lang/java/tools/src/test/java/org/apache/avro/tool/TestConcatTool.java

```diff
@@ -136,6 +137,38 @@ public void testConcat() throws Exception {
   }
 
   @Test
+  @Ignore
+  // theoretically this test should test the append scenario
+  // the problem is, FileSystem.append does not work for local filesystems
+  // (note: or maybe doing dev on Windows is the problem??)
+  // the best way to test is to write to HDFS and compare append output to the
+  // fresh output file generated by the plain concat operation writing to HDFS
```

Comment:
This all seems pretty daunting, so I'd be happy if you stat by filing a follow on issue that records the need to switch from LocalFileSystem to the mini-dfs cluster. That way you can reference the follow on here in the note about the limitations of LocalFileSystem. Then (if you're up for it!) you can handle the follow on (or someone else can if you'd rather sit that one out).